examples/schedules_construction/AMA_Advanced2024.py

```diff
@@ -0,0 +1,246 @@
+"""This file defines a AMA Advanced 2024 sequence using PyFligthCoach ManDef Classes and helper functions.
+   Author Angel Espinosa with help from Thomas David.
+"""
+
+from flightanalysis.definition import *
+from flightanalysis.elements import *
+from flightanalysis.scoring.criteria import *
+import numpy as np
+
+c45 = np.cos(np.radians(45))
+
+Adv24FC_def = SchedDef([
+
+    f3amb.create(ManInfo("Triang-roll", "trgle-roll", 3,Position.CENTRE,
+            BoxLocation(Height.TOP, Direction.UPWIND, Orientation.UPRIGHT),
+            BoxLocation(Height.MID)
+        ),[
+            MBTags.CENTRE,
+            f3amb.loop(r(-1/8)),
+            f3amb.line(),
+            f3amb.loop(r(-3/8)), 
+            centred(f3amb.roll(np.pi*2,line_length=2*107,padded=True)), 
+            f3amb.loop(-r(3/8)),
+            f3amb.line(),
+            f3amb.loop(-r(1/8)),
+            MBTags.CENTRE,
+        ], line_length=150),
+
+    f3amb.create(ManInfo("half square-roll", "hSl-roll", 2, Position.END,
+            BoxLocation(Height.TOP, Direction.UPWIND, Orientation.UPRIGHT),
+            BoxLocation(Height.MID)
+        ),[
+            f3amb.loop(-r(1/4)),
+            centred(f3amb.roll(np.pi,line_length=2*55,padded=True)),
+#            centred(f3amb.roll(np.pi,padded=True)),
+            f3amb.loop(r(1/4)),
+        ]),
+
+    f3amb.create(ManInfo(
+            "SLC-Rolls", "slc-rolls", k=4, position=Position.CENTRE, 
+            start=BoxLocation(Height.BTM, Direction.DOWNWIND, Orientation.UPRIGHT),
+            end=BoxLocation(Height.BTM)
+        ),[
+           MBTags.CENTRE,
+            f3amb.loop(r(1/8)),
+            f3amb.line(),
+            f3amb.loop(r(1/4)),
+            centred(f3amb.roll(np.pi,line_length=2*40,padded=True)),
+            f3amb.loop(-r(1/4)),
+            centred(f3amb.roll(np.pi,line_length=2*40,padded=True)),
+            f3amb.loop(r(1/4)),
+            f3amb.line(),
+            f3amb.loop(r(1/8)),
+            MBTags.CENTRE, 
+#        ], loop_radius = 30, line_length=80),
+        ], line_length=80),
+
+    f3amb.create(ManInfo(
+            "Figure 9", "fig9", k=3, position=Position.END, 
+            start=BoxLocation(Height.TOP, Direction.DOWNWIND, Orientation.UPRIGHT),
+            end=BoxLocation(Height.MID)
+        ),[
+            f3amb.loop(np.pi*2/4),
+            centred(f3amb.roll(np.pi,line_length=2*60,padded=True)),
+            f3amb.loop(np.pi*2*3/4),
+            f3amb.line(length = 100), 
+        ], loop_radius = 40),
+
+    f3amb.create(ManInfo(
+            "1/4 Rolls", "1/4Rolls", k=4, position=Position.CENTRE, 
```

Comment:
the short name (second name in maninfo) needs to be a valid attribute name in python, so you cant start it with a number or have a / in it. change this to 'rolls' or 'rCombo' or something like that.